manifests/params.pp

```diff
@@ -310,13 +316,23 @@
   }
 
   $server_jvm_java_bin = '/usr/bin/java'
-  $server_jvm_min_heap_size = '2G'
-  $server_jvm_max_heap_size = '2G'
+  $server_jvm_min_heap_size = '4G'
+  $server_jvm_max_heap_size = '4G'
```

Comment:
I had 1.1.something running with 1G of heap. What I want to avoid is to shift away from the situation that at the moment some VM with 1-2GB RAM (or even a RPi or Odroid C1 with a little swap) can be used to try out the usual all-in-one foreman+proxy+puppetmaster setup. When puppetmaster is allowed to already allocate some gigs of RAM only for startup, I'll walk away at that point from testing this on e.g. my laptop. It's quite clear that the CLJ/JVM approach will always need more RAM, but it would be ncie to have this as low as possible to work OK for small setups and document prominently ion the README how to raise this for bigger deployments.